src/components/binding/binding-panel.js

```diff
@@ -163,24 +163,24 @@ genotet.BindingPanel.prototype.initChrs_ = function() {
  * @private
  */
 genotet.BindingPanel.prototype.addTrack_ = function() {
-  var trackIndex = this.data.tracks.length - 1;
   var ui = this.container.find('#genes #tracks');
+  var trackIndex = ui.children().length - 1;
   var uiTrack = ui.find('#track-template').clone()
     .appendTo(ui)
     .attr('id', 'track-' + trackIndex)
     .show();
 
-  var genes = genotet.data.bindingGenes.map(function(gene) {
+  var genes = Object.keys(genotet.data.bindingGenes).map(function(gene) {
```

Comment:
rename bindingGenes to bindingFiles, which stores an array of binding file info.